package.json

```
@@ -8,7 +8,8 @@
     "url": "http://github.com/auth0/limitd-redis.git"
   },
   "scripts": {
-    "test": "trap 'docker-compose down --remove-orphans -v' EXIT; docker-compose up -d && NODE_ENV=test nyc mocha --exit"
+    "test-standalone": "CLUSTERED_ENV=false NODE_ENV=test nyc mocha --exit",
```

I'm not a huge fan of this way of running a subset of tests.

I think a better way would be to use mocha to filter for specific test file names. Mocha accepts a glob pattern for test files.

Not tested, but something like:
```suggestion
    "test-standalone": "NODE_ENV=test nyc mocha --exit '!**/*.clustermode.tests.js,**/*.standalone.tests.js'",
```

---

Fixed. Did it a bit different but same idea. Thanks for the suggestion